com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/HDDiffuseShadowDenoiser.cs

```diff
@@ -289,5 +352,65 @@ static void ExecuteDiffuseShadowSphereDenoiser(CommandBuffer cmd, DiffuseShadowS
             DiffuseShadowSphereDenoiserResources dssdResources = PrepareDiffuseShadowSphereDenoiserResources(distanceBuffer, noisyBuffer, intermediateBuffer, outputBuffer);
             ExecuteDiffuseShadowSphereDenoiser(cmd, dssdParams, dssdResources);
         }
+
+        class DiffuseShadowDenoiserSpherePassData
+        {
+            public DiffuseShadowSphereDenoiserParameters parameters;
+            public TextureHandle depthStencilBuffer;
+            public TextureHandle normalBuffer;
+            public TextureHandle distanceBuffer;
+            public TextureHandle noisyBuffer;
+            public TextureHandle intermediateBuffer;
+            public TextureHandle outputBuffer;
+        }
+
+        public TextureHandle DenoiseBufferSphere(RenderGraph renderGraph, HDCamera hdCamera,
+                            TextureHandle depthBuffer, TextureHandle normalBuffer,
+                            TextureHandle noisyBuffer, TextureHandle distanceBuffer,
+                            int kernelSize, Vector3 lightPosition, float lightRadius)
+        {
+            // Request the intermediate buffer we need
+            RTHandle intermediateBuffer = m_RenderPipeline.GetRayTracingBuffer(InternalRayTracingBuffers.RGBA3);
```

Comment:
Same as above (also, the format does not match the one in the render graph pass. Is that intended? Or is RGBA3 not what I think it is?)